File: Projekt_Python/rosliny/Trawa.py
from Roslina import Roslina

class Trawa (Roslina):
    def __init__(self, x, y,dane):
        if dane==None:
            self._x = x
            self._y = y
            self._symbol = 't'
            self._sila = 0
            self._wykonal_ruch = False
            self._inicjatywa = 0
            self._zycie = 0
            self._numer=0
        else:
            self._x = dane[0]
            self._y = dane[1]
            self._inicjatywa = dane[2]
            self._symbol = 't'
            self._sila = dane[3]
            self._wykonal_ruch = False
            self._zycie = dane[4]
            self._numer = dane[5]
        self._kolor= (0,255,0)

File: projekt_java/src/po/java/rosliny/Trawa.java
package po.java.rosliny;

import po.java.Roslina;

import java.awt.*;

public class Trawa extends Roslina {
    public Trawa (int x, int y)
    {
        this.x = x;
        this.y = y;
        this.symbol = 't';
        this.sila = 0;
        this.wykonal_ruch = false;
        this.zycie = 0;
        this.numer=0;
        this.kolor= Color.GREEN;
    }
    public Trawa(int []dane)
    {
        this.x = dane[0];
        this.y = dane[1];
        this.inicjatywa=dane[2];
        this.symbol = 't';
        this.sila = dane[3];
        this.wykonal_ruch = false;
        this.zycie = dane[4];
        this.numer=dane[5];
        this.kolor= Color.GREEN;
    }
}
